ecs-web/task-definitions/concourse_web_service.json

```diff
@@ -32,7 +32,7 @@
     "name"   : "concourse_web",
     "image"  : "${image}",
     "command": [
-      "concourse web ${vault_command_args}"
+      "PATH=$PATH:/usr/local/concourse/bin:/usr/local/bin concourse web ${vault_command_args}"
```

Comment:
Why overriding the PATH instead of using the full binary path directly? Are there other binaries there that concourse uses?

Reply:
because the `concourse` binary is located at `/usr/local/bin` for versions < 5 and at `/usr/local/concourse/bin` for versions >= 5. This command supports both versions.